src/components/error-boundary/error-boundary.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const ErrorBoundary = ({children, isEveryThingOk}) => {
```

Comment:
Can't find how you catch error. Look at the docs https://reactjs.org/docs/error-boundaries.html 
Also you can create a hook if you want